packyou/py3.py

```diff
@@ -11,7 +11,7 @@
 
 from importlib.abc import MetaPathFinder
 
-from packyou import find_module_in_cloned_report
```

Comment:
haha! I was working with reports. thanks for fixing this typo

Reply:
hehe yeah, any eta on cutting the new `0.1.6` release to `pypi`? I would love to help push that

Reply:
are you using python 2?

Reply:
nope